themes/gatsby-theme-wordpress-blog/gatsby-node.js

```diff
@@ -43,14 +43,79 @@ exports.createSchemaCustomization = async ({ actions }) => {
       url: String @proxy(from: "mediaItemUrl")
     }
 
-    type WpPost implements Node & BlogPost {
+    # custom type for blog posts
+    type WpBlogPost implements Node & BlogPost {
       id: ID!
       slug: String!
       title: String!
-      html: String! @proxy(from: "content")
+      html: String!
       excerpt: String! @sanitizeHTML
       date: Date!
-      image: Image @link(by: "id", from: "featuredImage.node.id")
+      image: Image @link
+      author: BlogAuthor @link(by: "parent.id")
+    }
+
+    # custom type for blog theme implementation
+    type WpBlogAuthor implements Node & BlogAuthor {
```

Comment:
Because WP is more opinionated with how it handles posts and users/authors, this creates new nodes for use in this theme